lib/buffers.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var Buffers = require('buffers');
+
+Buffers.prototype.skip = function(i) {
+  if (i === 0) {
+    return;
+  }
+
+  if (i === this.length) {
+    this.buffers = [];
+    this.length = 0;
+    return;
+  }
+
+  var pos = this.pos(i);
+  this.buffers = this.buffers.slice(pos.buf);
+  this.buffers[0] = new Buffer(this.buffers[0].slice(pos.offset));
+  this.length -= i;
+};
+
+module.exports = Buffers;
+
```

Comment:
This seems like a pretty useful function. Should we submit it to the global buffers module https://github.com/substack/node-buffers? I'd be happy to give it a go if that's the right thing to do.